### DISCUSSION TITLE
Domain still pointed at old site

### DISCUSSION THREAD

**Discussion by hicksticks** [vercel/community#1726] [Help] [answered]
### Summary

A domain I own is pointed tied to a vercel site that I do not own.   We have made a new version of the website, and want to point to it now.  I change the A record and CNAME in namescheap.com, but it still not working.   When I try to add it manully thru my vercel > View Domains, it gives me this error:

Domain is pending verification
Error:Domain conkme.com was added to a different project. Please complete verification to add it to this project instead

Is there something I can do without having access to the old site/project?

**Comment by patricioBoss** [comment]
i am also having this same issue, how was this resolved?

**Comment by swarnava** [comment, marked as answer]
You need to add the domain in new project > Setup the `TXT` record to transfer the Domain to your new projet to setup it. 

> Error:Domain conkme.com was added to a different project. Please complete verification to add it to this project instead
